mods/ts/weapons/otherweapons.yaml

```diff
@@ -99,6 +99,21 @@ SlimeAttack:
 			Concrete: 20
 		DamageTypes: Prone100Percent, TriggerProne, SmallExplosionDeath
 
+Tentacle:
+	ReloadDelay: 80
+	Range: 1c384
+	Report: floatk1.aud
+	Projectile: InstantHit
+	Warhead@1Dam: TargetDamage
+		Damage: 160
+		Versus:
+			None: 60
+			Wood: 45
+			Light: 90
+			Heavy: 55
+			Concrete: 0
```

Comment:
Default value is afaik 100, not 0. This should be replaced with a proper InvaildTargets: tag instead to make unit don't attack Structures. Also without InvaildTargets:, this doesn't make them don't attack anyway, just don't damage.